src/python/bot/tasks/fuzz_task.py

```diff
@@ -1536,3 +1539,106 @@ def execute_task(fuzzer_name, job_type):
   del testcase_file_paths
   del testcases_metadata
   utils.python_gc()
+
+
+def fuchsia_qemu_setup():
```

Comment:
i know this is a somewhat awkward place to put a platform-specific function, but platforms/fuchsia/device.py didn't seem quite right either; is there someplace where this would maek the most sense?

Reply:
I think device.py makes sense? Why do you disagree?

Reply:
eh.  not a super-strong rationale.  moved to device.py!